AntennaTracker/sensors.cpp

```diff
@@ -66,6 +66,21 @@ void Tracker::compass_cal_update() {
 }
 
 /*
+    Accel calibration
+*/
```

Comment:
most of this function could be in common code. We shouldn't be repeating new logic in each vehicle type like this.